worlds/aquaria/docs/setup_fr.md

```diff
@@ -116,3 +121,15 @@ pour vous assurer que votre fichier est exécutable:
 ```bash
 chmod +x aquaria_randomizer
 ```
+
+## Suivi automatique
+
+Aquaria permet un suivi complètement automatique des cartes du jeu dans le tracker.
+
+1. Téléchargez [Aquaria AP Tracker](https://github.com/palex00/aquaria-ap-tracker/releases/latest) et [PopTracker](https://github.com/black-sliver/PopTracker/releases/latest).
+2. Mettre le paquet du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.
```

Comment:
```suggestion
2. Mettre le fichier du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.
```

Reply:
Fichier sounds better than Paquet for me.

Reply:
Good for me. I did not know how to translate package.

Reply:
Fichier is technically the word for file in French but yeah

Reply:
I know. But it make sense.

Reply:
I dunno, feel like that might be confusing, I still feel like "dossier" is more fitting.

Of course, that's all small potatoes, it's basically fine as-is.

Reply:
From my somewhat brief jaunt into the English French dictionary on Linguee it says that zip file is “fichier compressé”

Reply:
Would everyone be okay with "fichier compressé"? @Cipocreep @tioui

Reply:
I mean I guess? It might just introduce more confusion, like, the poptracker pack doesn't need to be zipped (it can be, I'm pretty sure, but it doesn't NEED to be), I feel like either "fichier" or "dossier" alone are better.

Reply:
> Would everyone be okay with "fichier compressé"? @Cipocreep @tioui

Yes. Good form me.

Reply:
Yeah, it's tioui is right, let's not lose sleep over this

Reply:
100%